cccatalog-api/cccatalog/api/views/image_views.py

```diff
@@ -75,7 +76,10 @@ def get(self, request, format=None):
 
         search_index = 'search-qa' if qa else 'image'
         try:
-            results, page_count, result_count = search_controller.search(
```

Comment:
Could you undo the style changes in this file on lines 79-82 and 254-255? After that, I'll merge this PR. It looks good!

Reply:
I did not do any changes on lines 245-255, but I have reverted those on 79-82 in the latest commit